lib/widgets/priority_dropdown.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+import 'package:geeruh/theme.dart';
+
+final List<String> priorityLevel = <String>['Low', 'Medium', 'High', 'Top'];
+final List<Image> priorityImages = <Image>[
+  Image.asset('images/PriorityLow.png'),
+  Image.asset('images/PriorityMedium.png'),
+  Image.asset('images/PriorityHigh.png'),
+  Image.asset('images/PriorityTop.png'),
+];
+
+String? currentValue = priorityLevel[0];
+
+class PriorityDropdown extends StatefulWidget {
+  const PriorityDropdown({super.key, this.width = 150, this.heigth = 40});
+
+  final double width;
+  final double heigth;
+
+  @override
+  State<PriorityDropdown> createState() => _PriorityDropdown();
+}
+
+class _PriorityDropdown extends State<PriorityDropdown> {
+  double get width => widget.width;
+  double get heigth => widget.heigth;
```

Comment:
Uwagi do getterów w widgetach
1. Jeśli mamy widget prywatny (zaczynający się od "_") to są one do niczego, bo i tak nie zmieniamy tych wartości poza plikiem (w tym pliku w sumie też, bo nie jest tworzony explicite żaden obiekt)
2. Nie ma potrzeby ich ustawiać dla wygody (pisanie width zamiast widget.width), bo równie dobrze można sobie to zdefiniować na początku metody build, bez słowa kluczowego "getter"
3. Nie należy ich ustawiać dla szerokości i wysokości, bo są one i tak znane uprzednio przy inicjowaniu widgeta, więc wypada je raczej pamiętać, jeśli zachodzi taka potrzeba, w możliwie najwęższym kontekście